src/users/entities/user.entity.ts

```diff
@@ -1,21 +1,21 @@
 import { Roles } from 'src/types/enums/user-roles.enum';
 import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
 
-@Entity()
+@Entity({ name: 'users' })
 export class User {
   @PrimaryGeneratedColumn('uuid')
   id: string;
 
-  @Column()
+  @Column({ nullable: true })
```

Comment:
why you set all the fields to nullable ?

Reply:
I needed a new table with nullable columns to create a default admin

Reply:
its better to set some default values instead of set this fields to nullable